### PR TITLE
Add an icon to the call button to draw more attention

### LINKED ISSUE #2543: Display icon-loading on the right of the join-call button instead of changing text

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #2543

### CODE REVIEW

File: css/icons.scss [hunk 1]
```diff
@@ -80,6 +80,15 @@
 		@include icon-color('video', 'actions', 'E9322D', 1, true);
 	}
 
+	.icon-start-call,
+	.icon-incoming-call {
+		@include icon-color('video', 'actions', $color-primary-text, 1, true);
```
Comment (nickvergessen) [inline css/icons.scss]: Broken on dark mode due to https://github.com/nextcloud/server/issues/19565

**Comment by nickvergessen** [comment]
Please review @ma12-co

**Comment by danxuliu** [comment]
There are two minor style issues (loading icon image instead of pure CSS and default cursor on icon instead of pointer) that come from the server and will be fixed there (https://github.com/nextcloud/server/pull/19671 and https://github.com/nextcloud/server/pull/19672).

I guess those fixes will not be backported to _stable18_ in server, should they be fixed only for Talk 8 in https://github.com/nextcloud/spreed/pull/3008 (or a follow up)?

**Comment by nickvergessen** [comment]
> I guess those fixes will not be backported to stable18 in server

If it is not important enough to backport it, then we don't backport it in talk either

**Comment by danxuliu** [comment]
> > I guess those fixes will not be backported to stable18 in server
>
> If it is not important enough to backport it, then we don't backport it in talk either

It is not the same; the fixes in the server are not actually "needed", because it can be solved without problems in the apps (for example, as [it was done for the clipboard in Talk 7](https://github.com/nextcloud/spreed/blob/aaf289704ec72e078d7b99efe690d37d16567659/css/style.scss#L1017)). But in Talk 8 the issue will be visible in the call button. But OK, no fix for Talk 8, less work ;-)

**Pull request merged**